documentation/API_documentation/QoD_Latency_Bandwidth_User_Story.md

```diff
@@ -0,0 +1,10 @@
+| ***Pre-conditions*** |The preconditions are listed below:<br><ol><li>The Customer:BusinessManager and Customer:Administrator have been onboarded to the CSP's API platform.&nbsp;&nbsp;</li><li><span class="colour" style="color:var(--vscode-unotes-wysTableText)"><span class="font" style="font-family:var(--vscode-editor-font-family)"><span class="size" style="font-size:1em">The Customer:BusinessManager has successfully subscribed to the QoD product from the product catalog. &nbsp;&nbsp;</li><li><span class="colour" style="color:var(--vscode-unotes-wysTableText)"><span class="font" style="font-family:var(--vscode-editor-font-family)"><span class="size" style="font-size:1em">The Customer:Administrator has onboarded the Customer:User to the platform .</li><li><span class="colour" style="color:var(--vscode-unotes-wysTableText)"><span class="font" style="font-family:var(--vscode-editor-font-family)"><span class="size" style="font-size:1em">The Customer either owns the device subscriptions or has agreed to the terms and conditions of the CSP for managing consent of subscription owners.&nbsp;&nbsp;</li><li><span class="colour" style="color:var(--vscode-unotes-wysTableText)"><span class="font" style="font-family:var(--vscode-editor-font-family)"><span class="size" style="font-size:1em">The access token URL is known to the Customer:User to ensure secure means of API access. |
+| ***Activities/Steps*** | **Starts when:** The customer application server makes a POST request to the QoD API to create a new QoD session<br>**Ends when:** The customer application server makes a DELETE request to the QoD API, or the QoD session deletion was triggered automatically either because the customer specified duration has expired or the default expiration limit set by the QoD service provider has reached. |
+| ***Post-conditions*** | The specified App-Flow was provided stable throughput/latency and if callback url was specified, a QoD session termination notification was received at the end.  |
+| ***Exceptions*** | Several exceptions might occur during the QoD API operations<br>- Unauthorized: Not valid credentials (e.g. use of already expired access token).<br>- Invalid input: Not valid input data to invoke operation (e.g. IP address format not as expected).<br>- Conflict: Internal configuration policies didn’t allow for operations.<br>- Session-not-created: Due to other internal errors, QoS session could not be created<span class="Apple-converted-space">&nbsp;</span> |
```

Comment:
+: session-cannot-fulfill: Due to changes in network conditions, when the throughput cannot be fulfilled any more.

Reply:
That is not an exception but will be handled by the network.